common/bet.ts

```diff
@@ -25,8 +26,10 @@ export type Bet = {
   isAnte?: boolean
   isLiquidityProvision?: boolean
   isRedemption?: boolean
-
-  createdTime: number
+  // A record of each transaction that partially (or fully) fills the bet amount.
+  // I.e. A limit order could be filled by partially matching with several bets.
+  // Non-limit orders can also be filled by matching with multiple limit orders.
+  fills?: fill[]
```

Comment:
The bets subcollection is going to get pretty big with all the limit order and fills being stored...